storageservice/modules/files/file_client.bal

```diff
@@ -384,6 +430,49 @@ public isolated client class FileClient {
         }
     }
 
+
+    # Downloads a file from fileshare to a specified location as an Byte array
+    #
+    # + fileShareName - Name of the FileShare
+    # + fileName - Name of the file
+    # + azureDirectoryPath - Path of azure directory
+    # + return -  If successful, file content as a byte array. Else an error
+    @display {label: "Download File"}
+    remote isolated function getFileAsByteArray(@display {label: "File Share Name"} string fileShareName, 
+                                     @display {label: "Azure File Name"} string fileName,  
+                                     @display {label: "Azure Directory Path"} string? azureDirectoryPath = ()) 
+                                     returns @display {label: "Response"} byte[]|error {
+        string requestPath = azureDirectoryPath is () ? (SLASH + fileShareName + SLASH + fileName) : (SLASH 
```

Comment:
File separator? Doesn't Ballerina provide a const for this?

Reply:
@kalaiyarasiganeshalingam do we support this in `file` module?

Reply:
No, we don't have any public constant for this in the file module.